cypress/e2e/feeder-rss-import.cy.ts

```diff
@@ -0,0 +1,47 @@
+describe("Feeder RSS Import", () => {
+  before(() => {
+    cy.login(Cypress.env("TEST_PRX_USER"), Cypress.env("TEST_PRX_PASS"));
+    Cypress.config({ baseUrl: `https://${Cypress.env("FEEDER_HOST")}` });
+  });
+
+  it("imports an RSS feed", () => {
+    const now = new Date().toISOString();
+    const canary = `Acceptance Test: ${now}`;
+
+    cy.visit("/podcasts");
+    cy.contains("My Podcasts");
+
+    // create a new podcast
+    cy.get('a.btn-success[href="/podcasts/new"]').click();
+    cy.contains("New Podcast");
+    cy.get("#podcast_title").type(canary);
+    cy.get("#podcast_default_feed_attributes_itunes_category").select("Arts", { force: true });
+    cy.contains(".btn", "Create").click();
```

Comment:
Should this wait for a few milliseconds so there is time for the success message to appear?

Reply:
Hmm... shouldn't need to?  The next line `cy.contains("Podcast created")` will wait up to 4 seconds (the cypress default) for that text to appear.